backend/src/main/java/kg16/demo/web/LogController.java

```diff
@@ -0,0 +1,81 @@
+package kg16.demo.web;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+class LogController {
+
+    @GetMapping("/logs")
+    public List<Map<String, Object>> getLogs(@RequestParam(required = false) String deviceId,
```

Comment:
Create a DTO instead of a Map would be usefull here
[Using map vs DTO in spring-boot applicatoin ](https://stackoverflow.com/questions/59374898/using-map-vs-dto-in-spring-boot-applicatoin)